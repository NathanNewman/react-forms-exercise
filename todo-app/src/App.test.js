import React from "react";
import ReactDOM from "react-dom";
import { render, cleanup } from "@testing-library/react";
import App from "./App";

afterEach(cleanup);

// smoke test
it("renders without crashing", function() {
  render(<App />);
});

// snapshot
test("it matches snapshot", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
