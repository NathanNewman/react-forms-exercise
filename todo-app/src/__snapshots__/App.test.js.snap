// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <form>
        <label
          for="toDoInput"
        >
          Create New ToDo:
        </label>
        <input
          id="toDoInput"
          name="name"
          placeholder="ToDo"
          type="text"
          value=""
        />
        <button>
          Add to list
        </button>
      </form>
      <h2>
        ToDo List
      </h2>
      <div>
        <p>
          Code App 
          <button>
            X
          </button>
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
